### PR TITLE
Adds Prescription huds

### PR DESCRIPTION
**Pull request by Adrian16199**
## About the PR

WELCOME TO ANYONE WHO IS TUUUUUUUUUUNING IN.
THIS JUST IN, A WEIRD CAT HAS MADE A REVOLUTIONARY DISCOVERY AND ITS IN YOUR NEIGHBOUR HOOD!
IT WILL HELP THOSE 3 SPECIFIC PEOPLE WITH EYE PROBLEMS OR DEFFICIENCY (Not sure which one) 
WITH THEEEEEEEIR STRUGGLES!!
HE HAS GLUED TOGETHER AND ADJUSTED IT WITH CABLES, A MARVELOUS PIECE OF TECHNOLOGY WITH HIS GLASSES.
PREEESENTING!
Prescription medical hud!
=w=

(For legal reasons, this advertisement was not intended to mentally or physically harm anyone nor attempt to be legally set to court due to copyright or due to resulted mass death of cats.)
## Why / Balance

Yes.
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
image

## Breaking changes

**Changelog**

:cl:
- add: Added prescription medical hud and security hud.

### CODE REVIEW

**Comment by DangerRevolution** [comment]
Shouldn't this be in the vendor like the eyepatches?

**Comment by Adrian16199** [comment]
I mean, if the idea was to just present prescription medical hud that are *properly* build then I mean, I would need to make a different sprite for it and well, I just find this more funny that you would need to try really hard just to get these and its not like for some reason medical only gets medical hud eyepatches instead of normal ones but eee (shrug)
Even then, if its only intended to have 2 medical eyepatches in the vendor from wizden, addin 2 more but with just tag of prescription, would just make normal people take it incase the normal ones would be gone. At best, I would PR into upstream that medical huds would even appear in medical vendors instead of only the eyepatch one.

**Comment by DangerRevolution** [comment]
> I mean, if the idea was to just present prescription medical hud that are _properly_ build then I mean, I would need to make a different sprite for it and well, I just find this more funny that you would need to try really hard just to get these and its not like for some reason medical only gets medical hud eyepatches instead of normal ones but eee (shrug) Even then, if its only intended to have 2 medical eyepatches in the vendor from wizden, addin 2 more but with just tag of prescription, would just make normal people take it incase the normal ones would be gone. At best, I would PR into upstream that medical huds would even appear in medical vendors instead of only the eyepatch one.

Tbh thinking about it, this is probably best off upstreamed given it's QoL moreso than a unique Delta-V thing but honestly I think the sprite would pass. I honestly would've done it like park and just used the same sprite but renamed it "prescription medhud" or smth.

also the test error is fake :O

**Comment by Adrian16199** [comment]
Upstream doesnt have nearsighted trait.
By upstream, I meant specficly to even allow medhuds to be even avaible thro vendors for medical and not only those eyepatches that are just weird.

**Comment by DEATHB4DEFEAT** [comment]
I had only read the title and description and thought "it's funny having one eye being covered magically fix your eye issues, wonder if that's still happening" and then I saw the image and suddenly laughed seeing the mix of the two sprites, this looks great so far.

File: Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/hud.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesPrescriptionMedHud
+  name: prescription medical hud
+  description: A heads-up display that scans the humanoids in view and provides accurate data about their health status. This was put together with prescription glasses for anyone that wants to use the hud while maintaining their vision.
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/hud.yml]:
```suggestion
  description: A poorly done and rushed mix between half of a pair of prescription glasses and a medical HUD allowing you to see clearly out of one eye and view the health of people out of the other!
```

File: Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/hud.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  - type: Clothing
+    sprite: DeltaV/Clothing/Eyes/Hud/prescsechud.rsi
+  - type: ShowSecurityIcons
+  - type: Construction
+    graph: PrescriptionSecHud
+    node: prescsechud
+  - type: Tag
+    tags:
+    - HudSecurity
+    - GlassesNearsight
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/hud.yml]:
```suggestion
    - GlassesNearsight

```

File: Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/hud.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  - type: Tag
+    tags:
+    - HudMedical
+    - GlassesNearsight
+
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesPrescriptionHudSecurity
+  name: prescription security hud
+  description: A heads-up display that scans the humanoids in view and provides accurate data about their ID status and security records. This was put together with prescription glasses for anyone that wants to use the hud while maintaining their vision.
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/hud.yml]:
```suggestion
  description: A poorly done and rushed mix between half of a pair of prescription glasses and a security HUD allowing you to see clearly out of one eye and inspect the employee's ID and warning status in the other!
```

File: Resources/Prototypes/DeltaV/Recipes/Construction/Graphs/clothing/prescription_huds.yml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+              amount: 5
+              doAfter: 5
+            - tag: DrinkSpaceGlue
+              name: space glue
+              icon:
+                sprite: Objects/Consumable/Drinks/glue-tube.rsi
+                state: icon
+              doAfter: 5
+    - node: prescsechud
+      entity: ClothingEyesPrescriptionHudSecurity
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Recipes/Construction/Graphs/clothing/prescription_huds.yml]:
```suggestion
      entity: ClothingEyesPrescriptionHudSecurity

```

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]
Whatever IDE you use you need to get the [EditorConfig](https://editorconfig.org/#download) extension for it, you keep leaving no newlines at the end of files

**Comment by DEATHB4DEFEAT** [comment]
> I honestly would've done it like park and just used the same sprite but renamed it "prescription medhud" or smth.

We don't really have anyone too good at spriting in Park (and we didn't think of mixing the two existing ones) so we just left the sprites the same thinking it was really funny to have one eye being covered magically fix the other one, we definitely should have gotten sprites for it though, glad to see Adrian has made some.

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Pull request merged**